content/posts/gsoc-2021-announcement/index.md

```diff
@@ -0,0 +1,58 @@
+
+* Provide `rz-lang` and `rz-pipe` bindings so that the users can choose through scripts, as well.
+
+As an additional goal, I'll be working on improving the SVD parser and adding it to Rizin as a core plugin, so that it'll be shipped with Rizin and Cutter, as well. At the end, I also hope to write a good how-to article on how to add and use a profile
+
+I am engrossed by the wonderful feeling of community I have felt during contributing to Rizin. I have gained amazing insights and skills and feel very grateful to be working and learning more from one of the smartest, kindest and knowledgeable people I've ever known.
+
+Hoping to accomplish great things and have a really great summer. I wish the very best to all the folks who got in!
+
+## Pulak: Heap viewer for Cutter
+
+Hi, I am Pulak Malhotra from India. I am an undergraduate student and researcher at IIIT Hyderabad. GSoC provides me an excellent opportunity to work on real-world codebases, contribute to the open-source community, meet new people and learn new things. I am relatively new to reverse engineering. In the past, I enjoyed working on low-level systems in my university courses which drew me towards reverse engineering. Rizin's welcoming and helpful community is a significant factor that makes me want to contribute to the project.
+
+My GSoC project aims to deliver widgets that provide information about the heap state while debugging programs in Cutter. These widgets will give information regarding chunks in a heap and the bins in which free chunks are located. I also aim to deliver graphical visualization of the linked list of the arena bins. My project also includes refactoring the heap codebase in Rizin, so the heap allocator is dynamically selected based on binary. Currently, the heap allocators are compiled at compile time in Rizin. I will also try to add support for more heap allocators in Rizin.
+
+I contributed the Pull Request [#810](https://github.com/rizinorg/rizin/pull/810) as my microtask. I improved the output of existing heap-related commands like `dmh` and `dmhf` in this microtask, and I created new commands `dmhv` and `dmhd`. `dmhv` is similar to `dmh` command but provides extra information about the heap chunks and `dmhd` command provides concise information about the bins of main arena. At various points, while solving the issue, I felt lost, especially when I was not familiar with the codebase. To get over this, I would make small changes, rerun the code, and note the difference in output. While solving this issue, many members of Rizin gave me detailed advice and feedback and guided me at every step. While solving this issue, I came across some suggestions which I pursued further in PR [#912](https://github.com/rizinorg/rizin/pull/912) and issue [#1088](https://github.com/rizinorg/rizin/issues/1088).  Overall, it was a fantastic experience contributing to the Rizin.
```

Comment:
```suggestion
I contributed the Pull Request [#810](https://github.com/rizinorg/rizin/pull/810) as my microtask. I improved the output of existing heap-related commands like `dmh` and `dmhf` in this microtask, and I created new commands `dmhv` and `dmhd`. `dmhv` is similar to `dmh` command but provides extra information about the heap chunks and `dmhd` command provides concise information about the bins of main arena. At various points, while solving the issue, I felt lost, especially when I was not familiar with the codebase. To get over this, I would make small changes, rerun the code, and note the difference in output. While solving this issue, many members of Rizin gave me detailed advice and feedback and guided me at every step. I also came across some suggestions which I pursued further in PR [#912](https://github.com/rizinorg/rizin/pull/912) and issue [#1088](https://github.com/rizinorg/rizin/issues/1088).  Overall, it was a fantastic experience contributing to the Rizin.
```

Reply:
there are three times of "While solving this issue", you changed one. I'd try to change two so they all start different

Reply:
This change seems good to me.